DataFormats/TrackReco/interface/HitPattern.h

```diff
@@ -205,6 +206,12 @@ class HitPattern
     /// RPC region: 0 = barrel, 1 = endcap. Only valid for muon RPC patterns, of course.
     static uint16_t getRPCregion(uint16_t pattern);
```

Comment:
packing of GEM hit info into the hit pattern should be documented in the comment fields earlier in this file (~L35)